odata-protocol/11 Data Service Requests.md

```diff
@@ -1149,7 +1149,7 @@ matching the request after applying any
 [`$filter`](#SystemQueryOptionfilter) or
 [`$search`](#SystemQueryOptionsearch) system query options, formatted as
 a simple primitive integer value with media type `text/plain` without `charset` parameter,
-and the response body consisting only of octets `0x30` to `0x39`.
+and the response body following rule `int64value` without the sign, see [OData-ABNF](#ODataABNF).
```

Comment:
```suggestion
and the response body following rule `int64Value` without the sign, see [OData-ABNF](#ODataABNF).
```

Reply:
We could do that. On the other hand the [control information count (odata.count)](https://docs.oasis-open.org/odata/odata-json-format/v4.02/csd01/odata-json-format-v4.02-csd01.html#ControlInformationcountodatacount) is an `Edm.Int64` value, and I assumed the same to be true for the `/$count` value.